src/Cloud4RPi.cpp

```diff
@@ -0,0 +1,174 @@
+#include "Cloud4RPi.h"
+
+const int JSON_BUFFER_SIZE = 500;
```

Comment:
Seems like it's better move it to the .h file

Reply:
Move it into the header.

Reply:
done